### PR TITLE
[Hexagon] Use VTCM while scheduling conv2d op

### PR DESCRIPTION
**Pull request by abhikran-quic**
Use VTCM for E2E execution of conv2d to reduce tuning time.

cc: @csullivan @masahi

### CODE REVIEW

File: tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py [hunk 1]
```diff
@@ -248,8 +248,9 @@ def index_map_nchw32c_nchw8h8w32c(n_batch, channel, height, width, channel_32):
 
         # Add cache for input and output activation layout transform,
         # note that weight is already in correct layout
-        input_cache = sch.cache_read(conv2d_block, 0, "global")  # pylint: disable=unused-variable
-        output_cache = sch.cache_write(outer_block, 0, "global")  # pylint: disable=unused-variable
+        # pylint: disable=unused-variable
```
Comment (masahi) [inline tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py:251]: Put this at the top of the file (see other files disabling pylint) if you need it.

**Review by masahi — APPROVED** [review]
> to reduce tuning time

This test doesn't really tune anything, but does this change bring such non-trivial reduction?

**Review by csullivan — APPROVED** [review]

**Comment by abhikran-quic** [comment]
> > to reduce tuning time
> 
> This test doesn't really tune anything, but does this change bring such non-trivial reduction?

Yes. I saw the test execution time to come down to 25 mins instead of 40 mins.

File: tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py (hunk 1 above)
Comment (abhikran-quic) [reply tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py:251]: This was needed only for `input_cache` and `output_cache` variables. I will move it to the top in next PR.

**Pull request merged**